kolibri/utils/cli.py

```diff
@@ -178,6 +182,9 @@ def __init__(self, *args, **kwargs):
         super(KolibriDjangoCommand, self).__init__(*args, **kwargs)
 
     def invoke(self, ctx):
+        # Check if the current user is the kolibri user when running kolibri from Debian installer.
```

Comment:
I think we probably want to add this to both the `KolibriCommand` above and the `KolibriDjangoCommand`, as both presume the ability to access the `KOLIBRI_HOME` dir.

Can probably do a small refactor and have `base_params` and then `initialize_params` as additional to the `base_params` here.

Reply:
thank you! I will add it to KolibriCommand as well.

sorry I'm a bit confused about `base_params` you mentioned here. What is this for?

Reply:
for `base_params`, do you mean `debug_option` and `noinput_option`?

Reply:
Yes - the two args that are used in both cases (I didn't bother defining a list of them before because there was only one).

Reply:
makes sense to me. thank you! I will create `base_params` for those two options.